conf.py

```diff
@@ -1361,14 +1362,31 @@
 BODY_END = """
 <script src="/assets/js/tipuesearch_set.js"></script>
 <script src="/assets/js/tipuesearch.js"></script>
+<script src="/assets/js/jquery.watermark.js"></script>
 <script>
 $(document).ready(function() {
+"""
+BODY_END += """
     $('#tipue_search_input').tipuesearch({
         'show': 5,
         'mode': 'json',
         'contentLocation': '/assets/js/tipuesearch_content.json',
         'showUrl': false
     });
+"""
+if WEBSITE_PREVIEW:
+    BODY_END += """
+$('body').watermark({
+    texts : ["Preview Website", "Mate Desktop", "PREVIEW_COMMITID"],
```

Comment:
I know it's just a preview, but shouldn't we use "MATE Desktop"? :confused:

Reply:
Ok, has been modified.